plugins/c8y_firmware_plugin/src/common.rs

```diff
@@ -0,0 +1,61 @@
+use crate::download::DownloadFirmwareStatusMessage;
```

Comment:
I'd merge the content of this module and the `child_device.rs` module into a single `messages.rs` module as most of the stuff in these crates are about the message definitions. Having a separate `child_device.rs` module doesn't make much sense here either as this plugin is **only** about child devices.